### DISCUSSION TITLE
I accidentally discovered a GitHub bug or something like that, that can be used by hackers

### DISCUSSION THREAD

**Discussion by ATreeShine** [community/community#80980] [Other Feature Feedback, Questions, & Ideas] [answered]
### New bug or something

Bug

### How it went

So, last day I was signing up for a new account from my other computer, but I accidentally entered the wrong email without noticing it. When GitHub sent me the code, I went to "change email" option and from their I could search any user and then follow them. This can be used by hackers to get unlimited followers.

**Comment by Dheeraj-02NK** [comment]
Yes, But you can only use only those emails that u have access for, rest u cannot.
Pls accept this as answer

**Reply by ATreeShine** [reply]
No you can also use those emails that you don't have access to I can easily use example@gmail.com even if I don't have access to it

**Reply by Dheeraj-02NK** [reply]
Then where are you getting the code ?

**Reply by ATreeShine** [reply]
You don't need to verify the account to follow users

**Reply by Dheeraj-02NK** [reply, marked as answer]
![Screenshot 2023-12-20 122059](https://github.com/community/community/assets/95096072/50dfeec5-773b-4dc9-b5fc-7d2e45d33648)

GitHub will automatically delete unverified account once in a while, So using fake followers and likes will stay temporary
Pls refer the above image.

It's also possible that GitHub is aware of this aspect but might consider it as part of their commitment to allowing users some flexibility in experimenting with the platform.

**Comment by Dheeraj-AI007** [comment]
This should be fixed by github asap

**Reply by ATreeShine** [reply]
I need bounty
